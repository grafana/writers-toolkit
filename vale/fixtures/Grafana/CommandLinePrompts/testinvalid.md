Don't use:

- ```bash
  $ command
  ```
- ```bash
  # sudo
  ```
- ```console
  $ command
  ```
- ```console
  # sudo
  ```
