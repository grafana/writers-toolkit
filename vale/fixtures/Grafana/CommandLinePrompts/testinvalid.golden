testinvalid.md:4:3:Grafana.CommandLinePrompts:Don't add `$` or `#` as prompts before commands. Make it easy for users to copy and paste commands. Also, don't use `#` to include comments in commands. That explanation should be outside of the code block. 
testinvalid.md:7:3:Grafana.CommandLinePrompts:Don't add `$` or `#` as prompts before commands. Make it easy for users to copy and paste commands. Also, don't use `#` to include comments in commands. That explanation should be outside of the code block. 
testinvalid.md:10:3:Grafana.CommandLinePrompts:Don't add `$` or `#` as prompts before commands. Make it easy for users to copy and paste commands. Also, don't use `#` to include comments in commands. That explanation should be outside of the code block. 
testinvalid.md:13:3:Grafana.CommandLinePrompts:Don't add `$` or `#` as prompts before commands. Make it easy for users to copy and paste commands. Also, don't use `#` to include comments in commands. That explanation should be outside of the code block. 
